aws-android-sdk-cognitoauth/src/androidTest/java/com/amazonaws/mobileconnectors/cognitoauth/LocalDataManagerIntegrationTest.java

```diff
@@ -403,30 +405,17 @@ private void verifySharedPreferencesForCachedSession() {
         assertNotNull(sharedPreferencesForAuth.getString(lastAuthUserKey, null));
     }
 
-    private void verifySharedPreferencesForCachedState(AWSKeyValueStore awsKeyValueStore, String proofKeyValue, String scopesValue) {
+    private void verifySharedPreferencesForCachedState(AWSKeyValueStore awsKeyValueStore, String proofKeyValue, Set scopesValue) {
         assert sharedPreferencesForAuth.getAll().keySet().size() > 0;
         assert sharedPreferencesForAuthEncryptionMaterials.getAll().keySet().size() > 0;
 
-        assertEquals(proofKeyValue,
-                LocalDataManager.getCachedProofKey(awsKeyValueStore, InstrumentationRegistry.getTargetContext(), "key"));
-        assertEquals(scopesValue == null ? new HashSet<String>() : setFromString(scopesValue),
-                LocalDataManager.getCachedScopes(awsKeyValueStore, InstrumentationRegistry.getTargetContext(), "key"));
-    }
+        String cachedProofKey = LocalDataManager.getCachedProofKey(awsKeyValueStore, InstrumentationRegistry.getTargetContext(), "key");
+        Set cachedScopes = LocalDataManager.getCachedScopes(awsKeyValueStore, InstrumentationRegistry.getTargetContext(), "key");
```

Comment:
(Same here, `rawtypes` can be resolved by adding template param)

Reply:
Welp, at least I'm consistent in my bumbling. Thanks!